library/src/scala/quoted/QuoteContext.scala

```diff
@@ -81,8 +81,10 @@ trait QuoteContext { self: internal.QuoteUnpickler & internal.QuoteMatching =>
    *  ```none
    *
    *  +- Tree -+- PackageClause
-   *           +- Import
-   *           +- Statement -+- Definition --+- ClassDef
+   *           |
+   *           +- Statement -+- Import
```

Comment:
@nicolasstucki updating to put `Import` and `Export` under `Statement` as the types are defined

Reply:
Ok